iib/web/models.py

```diff
@@ -1543,6 +1544,7 @@ class RequestCreateEmptyIndex(Request, RequestIndexImageMixin):
         'polymorphic_identity': RequestTypeMapping.__members__['create_empty_index'].value
     }
     build_tags = None
+    fbc = None
 
```

Comment:
Since it will receive a boolean wouldn't it be better to assign `false` as default?

Or do we need to have a differentiation between `unset` and `bool`?

Reply:
yeah, I can change that